src/Corax/IndexEntryBuilder.cs

```diff
@@ -724,10 +773,34 @@ internal static (int, bool) GetMetadataFieldLocation(Span<byte> buffer, int fiel
 
                     throw new NotSupportedException($"The type {nameof(T)} is unsupported.");
                 }
+
+                if ((type & IndexEntryFieldType.Special) != 0)
+                {
+                    //<type><lat><long><amount_of_geohashes><pointer_to_string_length_table><geohash>
```

Comment:
why do we have amount of geo hashes? Shouldn't we have just one?

Reply:
Yes, I've changed the design for point:
Single:
`[header][lat:double][lon:double][geohash_lvl:int][geohash:string]`

List:
`<type:byte><extended_type:byte><amount_of_items:int><geohashLevel:int><geohash_ptr:int>
            //<longitudes_ptr:int><latitudes_list:double[]><longtitudes_list:double[]><geohashes_list:bytes[]>`

`Extended type` will disappear after updating to Federico PR where `type` is changed into `ushort` from `byte`